crates/sui-indexer-alt-reader/src/metrics.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::sync::Arc;
+
+use prometheus::{
+    register_histogram_with_registry, register_int_counter_with_registry, Histogram, IntCounter,
+    Registry,
+};
+
+/// Histogram buckets for the distribution of latency (time between receiving a request and sending
+/// a response).
```

Comment:
`time between receiving a request and sending a response` seems describing JSPNRPC server, change the wording to be more reader-related?